### ISSUE TITLE
HonDash not communicating with Kpro v4.5

### ISSUE THREAD

**Issue by integraK20** [pablobuenaposada/HonDash#255]
HonDash will not read and display info from kpro v4.5

**Comment by BFCE** [comment]
Same issue, bottom of the screen reads K-Pro v0.00, all the values read 0

**Comment by 736NV1** [comment]
> could you specify raspberry model and revision?

Hello, the same thing happens to me, everything stays at 0. My raspberry pi 4b+ 2gb year 2018. Kpro Updated to the latest version.

**Comment by dylanmilesmsu** [comment]
Just came back to this project after like a year. Before I tried it on a aml-s905x-cc, I got it running but all the values read zero. Today I set it up on my Pop!_OS laptop, got the correct python version running, got all the dependencies working, I don't see any errors, but yet it won't communicate with my ECU. Weather I plug my USB in before starting it or after, it doesn't matter, it reads that I have KPro version 0, 0 rpm, 0 coolant temp, 0 everything.

I think this project is just busted and does not work anymore with newer OSs or newer hondatas.
